py-polars/polars/dataframe/frame.py

```diff
@@ -226,9 +226,10 @@ class DataFrame:
         Whether to interpret two-dimensional data as columns or as rows. If None,
         the orientation is inferred by matching the columns and data dimensions. If
         this does not yield conclusive results, column orientation is used.
-    infer_schema_length : int, default None
+    infer_schema_length : int, default N_INFER_DEFAULT
```

Comment:
This does not render as `100`. Better just say `int or None` and do not mention the default.
```suggestion
    infer_schema_length : int or None
```

Reply:
Hi, thanks for reviewing.

I left it as draft because there were a couple of open points in the [issue](https://github.com/pola-rs/polars/issues/14232#issuecomment-1925438048), namely `str.json_decode`, `_scan_ndjson` have different behavior and I am not sure if they should be updated as well.

If not, I will just adjust these descriptions and set it ready for review.